src/test/scala/mesosphere/marathon/tasks/TaskOpFactoryImplTest.scala

```diff
@@ -159,11 +155,9 @@ class TaskOpFactoryImplTest extends MarathonSpec with GivenWhenThen with Mockito
 
     def normalApp = MTH.makeBasicApp()
     def residentApp = MTH.appWithPersistentVolume()
-    def normalLaunchedTask(appId: PathId, volumeIds: LocalVolumeId*) = MTH.mininimalTask(appId).copy(launched = Some(MTH.taskLaunched))
-    def residentReservedTask(appId: PathId, volumeIds: LocalVolumeId*) =
-      MTH.residentReservedTask(appId, volumeIds: _*)
-    def residentLaunchedTask(appId: PathId, volumeIds: LocalVolumeId*) =
-      MTH.residentReservedTask(appId, volumeIds: _*).copy(launched = Some(MTH.taskLaunched))
+    def normalLaunchedTask(appId: PathId) = MTH.mininimalLaunchedTask(appId.toString)
```

Comment:
BTW, I do not think that these forwarding methods help. Just added noise, in my opinion.